canonical-web-and-design/anbox-cloud.io
# WIP Anbox setup updates

## Done

- Updated package.json
- `/Docs` test

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8043/
- Make sure that the github check `python-test` passes or do `python3 -m unittest discover tests`
